src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,60 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Class to maintain the deadline tasks entered by user.
+ */
+public class Deadline extends Task {
+    protected LocalDate deadline;
+
+    protected LocalTime endTime;
+
+    /**
+     * Constructor for Deadline class.

```

Comment:
Perhaps there should be an empty line between description and parameter section.